com.unity.render-pipelines.universal/ShaderLibrary/SSAO.hlsl

```diff
@@ -319,63 +375,58 @@ float4 SSAO(Varyings input) : SV_Target
 // Geometry-aware separable bilateral filter
 half4 Blur(float2 uv, float2 delta) : SV_Target
```

Comment:
We could easily need this for some other effect too. Can we generalize it in a meaningful way? Or break into "building block" functions?

Reply:
I think that's a good idea.
My next SSAO PR will tackle different Blur Options, so I'll split the Blurring code into a separate file and see if I can be more general.